network/p2p/gossip/proto/message.proto

```diff
@@ -0,0 +1,14 @@
+syntax = "proto3";
```

Comment:
Can we move the proto definition + generated code into the `proto` folder with the rest of the proto definitions? That way `buf` will publish this proto with the rest (and it will be auto-generated with the rest)

Reply:
Chatted about this offline but this makes it easier for us to eventually refactor out the protobuf definitions.